.travis/build.sh

```diff
@@ -43,8 +43,51 @@ cat > pytest.ini << EOF
 addopts=-p no:logging --color=no --force-flaky
 EOF
 
+if [ "$TARGET_HOST" == "arm-linux-gnueabihf" ] || [ "$TARGET_HOST" == "arm-linux-gnueabihf" ]
```

Comment:
Why are we ORing two identical checks here? My guess is the second should be `aarch64`?